text/0097-log-data-model.md

```diff
@@ -1002,6 +1003,326 @@ span_id          | string             | The span ID within the trace associated
+  </tr>
+  <tr>
+    <td>error.code</td>
+    <td>string</td>
+    <td>Error code describing the error.</td>
+    <td>attributes[“error.code”]</td>
+  </tr>
+  <tr>
+    <td>error.id</td>
+    <td>string</td>
+    <td>Unique identifier for the error.</td>
+    <td>attributes[“error.id”]</td>
+  </tr>
+  <tr>
+    <td>error.message</td>
+    <td>string</td>
+    <td>Error message.</td>
+    <td>attributes[“error.stack_trace”]</td>
+  </tr>
+  <tr>
+    <td>error.stack_trace</td>
+    <td>string</td>
+    <td>The stack trace of this error in plain text.</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>host.architecture</td>
+    <td>string</td>
+    <td>Operating system architecture</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>host.domain</td>
+    <td>string</td>
+    <td>Name of the domain of which the host is a member.
+
+For example, on Windows this could be the host’s Active Directory domain or NetBIOS domain name. For Linux this could be the domain of the host’s LDAP provider.</td>
+
+<td>**resource</td>
+  </tr>
+  <tr>
+    <td>host.hostname</td>
+    <td>string</td>
+    <td>Hostname of the host.
+
+It normally contains what the hostname command returns on the host machine.</td>
+
+<td>resource[“host.hostname”]</td>
+
+  </tr>
+  <tr>
+    <td>host.id</td>
+    <td>string</td>
+    <td>Unique host id.</td>
+    <td>resource[“host.id”]</td>
+  </tr>
+  <tr>
+    <td>host.ip</td>
+    <td>Array of string</td>
+    <td>Host IP</td>
+    <td>resource[“host.ip”]</td>
+  </tr>
+  <tr>
+    <td>host.mac</td>
+    <td>array of string</td>
+    <td>MAC addresses of the host</td>
+    <td>resource[“host.mac”]</td>
+  </tr>
+  <tr>
+    <td>host.name</td>
+    <td>string</td>
+    <td>Name of the host.
+
+It may contain what hostname returns on Unix systems, the fully qualified, or a name specified by the user. </td>
+
+<td>resource[“host.name”]</td>
+
+  </tr>
+  <tr>
+    <td>host.type</td>
+    <td>string</td>
+    <td>Type of host.</td>
+    <td>resource[“host.type”]</td>
+  </tr>
+  <tr>
+    <td>host.uptime</td>
+    <td>string</td>
+    <td>Seconds the host has been up.</td>
+    <td>?</td>
+  </tr>
+  <tr>
+    <td>service.ephemeral_id
+
+</td>
+    <td>string</td>
+    <td>Ephemeral identifier of this service</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>service.id</td>
+    <td>string</td>
+    <td>Unique identifier of the running service. If the service is comprised of many nodes, the service.id should be the same for all nodes.</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>service.name</td>
+    <td>string</td>
+    <td>Name of the service data is collected from.</td>
+    <td>resource["service.name"]</td>
+  </tr>
+  <tr>
+    <td>service.node.name</td>
+    <td>string</td>
+    <td>Specific node serving that service</td>
+    <td>resource[“service.instance.id”]</td>
+  </tr>
+  <tr>
+    <td>service.state</td>
+    <td>string</td>
+    <td>Current state of the service.</td>
+    <td>**resource</td>
```

Comment:
If the state can change over the lifetime of the resource than it cannot be in the `resource`.